.github/workflows/regression_tests_docker.yml

```diff
@@ -0,0 +1,56 @@
+name: Run Regression Tests on Docker
+
+on:
+  push
+    # run every day at 5:15am
+  #schedule:
+  #  - cron:  '15 5 * * *'
+
+concurrency:
+  group: ci-cpu-${{ github.workflow }}-${{ github.ref == 'refs/heads/master' && github.run_number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  docker-regression:
+    strategy:
+      fail-fast: false
+      matrix:
+        hardware: [ubuntu-20.04, [self-hosted, regression-test-gpu]]
```

Comment:
why are there 2 github actions? This matrix seems to indicate that you're running both GPU and CPU tests here?